unleash-yggdrasil/src/lib.rs

```diff
@@ -612,6 +615,7 @@ pub struct VariantDef {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub payload: Option<Payload>,
     pub enabled: bool,
+    pub feature_enabled: bool,
 }
 
 impl Default for VariantDef {
```

Comment:
I think the default impl here is only used for tests but it's now definitely dangerous to use. Can we move this into the test mod so that it's no longer exposed